src/AWSS3.jl

```diff
@@ -217,7 +234,36 @@ function s3_exists(
         end
     end
 end
+s3_exists_versioned(a...; kw...) = s3_exists_unversioned(global_aws_config(), a...; kw...)
```

Comment:
```suggestion
s3_exists_versioned(a...; kw...) = s3_exists_versioned(global_aws_config(), a...; kw...)
```

Reply:
actually if these are internal, we can just delete these methods and require a config is always passed